WordPress/Sniff.php

```diff
@@ -1514,4 +1514,63 @@ public function get_function_call_parameter( $stackPtr, $param_offset ) {
 		return $parameters[ $param_offset ];
 	}
 
+	/**
+	 * Check if a content string contains a specific html open tag.
+	 *
+	 * {@internal For PHP 5.3+ this is straightforward, just check if $content
+	 * contains the tag.
+	 * PHP 5.2 however, creates a seperate token for `<s` when used in inline HTML,
```

Comment:
> Why did the tokenizer create a separate token in PHP 5.2?

See https://bugs.php.net/bug.php?id=48446:
> It stops at "<s" to avoid a long PHP opening tag (e.g. <script language="php">,
etc.) from being taken as inline HTML.

> The good news, however, is that a new scanner is used for PHP 5.3, and as of a few weeks ago (5.3.0 RC2), it now works as you'd expect.